src/sknnr/transformers/_standard_scaler_dof.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+from sklearn.preprocessing import StandardScaler
+
+
+class StandardScalerWithDOF(StandardScaler):
+    def __init__(self, ddof=0):
+        super().__init__()
+        self.ddof = ddof
+
+    def fit(self, X, y=None, sample_weight=None):
+        scaler = super().fit(X, y, sample_weight)
+        scaler.scale_ = np.std(np.array(X), axis=0, ddof=self.ddof)
```

Comment:
Nit: I think we should probably use [asarray](https://numpy.org/doc/stable/reference/generated/numpy.asarray.html) instead. I noticed that's what `sklearn` uses before running `np.nanstd` in one of their [utility functions](https://github.com/scikit-learn/scikit-learn/blob/364c77e047ca08a95862becf40a04fe9d4cd2c98/sklearn/preprocessing/_data.py#L223) for `StandardScaler`.

As far as I can tell the only difference is that `asarray` doesn't copy by default, so maybe a slight performance improvement.

Reply:
Good catch.  I've switched to `asarray` here and then also on `CCATransformer.fit` to make those tests pass.

Reply:
Great, thanks!